### ISSUE TITLE
Convert code to build transitions to Python module

### ISSUE THREAD

**Issue by crvernon** [IMMM-SFA/clm_demeter#6]
Convert @chenminbnu code to build transitions to Python module in this package.

**Comment by crvernon** [comment]
@evasinha We can discuss where this package should go once you get the original code.

**Comment by evasinha-pnnl** [comment]
Based on notes received from Min, the code for developing transition priorities was not developed. Should the three of us have a short call to discuss whether or not a code should be developed and who will take the lead on it?

**Comment by chenminbnu** [comment]
I agree. Sorry for that I didn't develop a code for this. I think I could contribute a flow chart or something like that to make the steps clear, and maybe @evasinha or @crvernon can prepare the python code? Sorry I am not familiar with python programming.

**Comment by evasinha-pnnl** [comment]
Min (@chenminbnu), I am happy to take the lead on it. A flow chart will be extremely helpful.

**Comment by chenminbnu** [comment]
@evasinha OK. I will develop the chart and send it to you later this week -- hopefully in the middle.

**Comment by chenminbnu** [comment]
@evasinha Thanks for leading on it!
